src/almanack/metrics/data.py

```diff
@@ -1182,3 +1195,71 @@ def find_doi_citation_data(repo: pygit2.Repository) -> Dict[str, Any]:
                 LOGGER.warning(f"Error during OpenAlex exact DOI lookup: {e}")
 
     return result
+
+
+def compute_sustainability_score(
+    table: List[Dict[str, Union[int, float, bool]]]
+) -> float:
+    """
+    Computes a sustainability score by normalizing
+    numeric metrics and incorporating boolean metrics.
+    Adjusts for metrics that should be inversely
+    proportional to the sustainability score.
+
```

Comment:
This paragraph formatting smells a bit strange to me. Please confirm

Reply:
Thanks - I'll rework this so it's more clear.